Gemfile.lock

```diff
@@ -64,7 +64,7 @@ GEM
     multi_json (1.10.1)
     nokogiri (1.6.4)
       mini_portile (~> 0.6.0)
-    pg (0.17.1)
+    pg (0.18.4)
```

Comment:
Hey -- our production version is locked on 0.17, I know have some dev issues with PG 0.17, but we're going to have to keep the PROD version static. Can you revert this, and we'll test your work on staging to make sure nothing you did is pg 0.17 specific.